invent-code-0/main.cpp

```diff
@@ -6,12 +6,14 @@ int main() {
   int price_avocado;
   int price_tomato;
   int num_avocados;
-  int num_tomatoes;
-  // Step 1: Prompt the user to input the number of tomatoes and avocados required.
+  int num_tomatoes,total_cost;
```

Comment:
why is total_cost declared here? you  can declare total cost below in step 3 directly